pkg/sparrow/run.go

```diff
@@ -141,6 +143,49 @@ func (s *Sparrow) ReconcileChecks(ctx context.Context) {
 	}
 }
 
+// updateCheckTargets updates the targets of a check with the
+// global targets. The targets are merged per default, if found in the
+// passed config.
+func (s *Sparrow) updateCheckTargets(cfg any) any {
+	if cfg == nil {
+		return nil
+	}
+
+	checkCfg, ok := cfg.(map[string]any)
+	if !ok {
+		return checkCfg
+	}
+	if _, ok = checkCfg["targets"]; !ok {
+		return checkCfg
+	}
+
+	actuali, ok := checkCfg["targets"].([]any)
+	if !ok {
+		return checkCfg
+	}
+
+	var actual []string
+	for _, v := range actuali {
+		actual = append(actual, v.(string))
```

Comment:
This could error if v is not a string

Reply:
Thanks for the catch 👍 Currently, all our `targets` are string slices, but still I'll do the due diligence and check if the target is a string as well.

Reply:
I've added the check and an empty slice check as well. Hopefully this function will be thrown out once we refactor the codebase next year, as editing a generic `any` data structure is borderline irresponsible. This is the simplest way ATM to implement the desired feature, without having to undergo a heavy refactoring.

The refactoring comes next year. Until then, we should make do with this small function, which can be replaced if need be.